src/core/main.c

```diff
@@ -1595,7 +1615,9 @@ static int become_shutdown(
                 watchdog_timer = arg_kexec_watchdog;
 
         /* If we reboot or kexec let's set the shutdown watchdog and tell the
-         * shutdown binary to repeatedly ping it */
+         * shutdown binary to repeatedly ping it.
+         * Disable the pretimeout watchdog, as we will not ping it from the shutdown binary. */
```

Comment:
i don't grok this comment btw? i assume that there was no separate "pretimeout" watchdog, i.e. not something you have to separately ping, is there?

Reply:
yeah it's not a separate ping, but it affects how often it happens, I'll reword